test/connection_init_test.exs

```diff
@@ -0,0 +1,40 @@
+defmodule RigInboundGatewayWeb.ConnectionInitTest do
+  use ExUnit.Case, async: true
+
+  require Logger
+
+  alias HTTPoison
+  alias Rig.Connection.Codec
+
+  @event_hub_http_port Confex.fetch_env!(:rig, RigInboundGatewayWeb.Endpoint)[:http][:port]
+
+  describe "Initialize connection" do
+    test "then connect to it" do
```

Comment:
the string (test name) should say what is expected. ex_unit doesn't lend itself very well to that given-when-then style of using describe blocks, as you cannot nest test code sufficiently. instead, use describe blocks sparingly and rely on the test's name/description to signal the test's intent. For example:

```elixir
test "An SSE connection can be initialized in advance." do
  ...
end
```